triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -192,10 +192,10 @@ final class Converter(
     }
 
   private[this] val fromCreatedEvent: CreatedEvent => Either[String, SValue] =
-    if (triggerDef.version < Trigger.Version.`2.6`) {
+    if (triggerDef.version < Trigger.Version.`2.5.0`) {
```

Comment:
This one have been improperly updated in [https://github.com/digital-asset/daml/pull/15902/files#diff-b20822516ed6db8d0cf27085397648f26ba15e406427d4457732f39fc50e14e5](#15902)